packages/components/dialog/src/use-dialog.ts

```diff
@@ -41,9 +42,12 @@ export const useDialog = (
     isNumber(props.width) ? `${props.width}px` : props.width
   )
 
+  const globalConfig = useGlobalConfig('namespace')
+  const namespace = computed(() => globalConfig.value || 'el')
```

Comment:
Can we export `defaultNamespace` and use like this?
```ts
import { defaultNamespace } from '@element-plus/hooks'
```

```suggestion
  const globalConfig = useGlobalConfig('namespace', defaultNamespace)
```

Reply:
Or to reduce duplicate code, we are still better off using hooks functions.